test/utils.sh

```diff
@@ -1199,3 +1199,40 @@ retry() {
       sleep "${interval}"
   done
 }
+
+# This function will be used by tasks in tekton-integration-catalog
+# If the image is an index (multi-arch), replace the digest with the 0th manifest's digest; if it's a single-arch image manifest, leave it unchanged.
+resolve_arch_specific_digest() {
```

Comment:
I think that the name of this function is also misleading. I would interpret it to require callers to also pass the desired architecture. This function more accurately gets the 0th manifest handling if the input is an image manifest or image index.